components/brave_wallet/browser/brave_wallet_service.cc

```diff
@@ -666,7 +666,7 @@ void BraveWalletService::OnGetImportInfo(
           [](ImportFromCryptoWalletsCallback callback,
              size_t number_of_accounts, KeyringController* keyring_controller,
              bool is_valid_mnemonic) {
-            if (number_of_accounts > 1) {
+            if (is_valid_mnemonic && number_of_accounts > 1) {
```

Comment:
error message for callback shouldn't be none when mnemonic is not valid

Reply:
we should add a new type of error and error message, otherwise no matter how many times users import from MetaMask, it would always failed with unknown reason in this case.

Reply:
Fixed in rebase